app/controllers/chores_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ChoresController < ApplicationController
+  def create
+    begin
+      #hideout_id = params[:payload]
+      params.require(%i[title description hideout_id assignee_email due_date])
+      assignee_id = User.find_by(id: params[:assignee_email])
+      chore = Chore.create(title: params[:title], description: params[:description], hideout_id: params[:hideout_id], assignee_id: params[:assignee_id], due_date: params[:due_date])
```

Comment:
The `hideout_id` should be accessed by accessing doing `payload = params[:payload]` and then `payload[:hideout_id]` since its added by the access token parsing. Make sure to pull from main before trying this.